src/search/components/SearchResultItem.tsx

```diff
@@ -64,14 +65,38 @@ const SearchResultItem: FunctionComponent<SearchResultItemProps> = ({
 			/>
 		);
 	};
+	const checkIfResultIsNotAnItem = (result: Avo.Search.ResultItem): boolean => {
+		return result.administrative_type === ('collectie' || 'bundel' || 'opdracht');
```

Comment:
this doesn't work. you should think about this from an order of execution perspective. The thing between the parenteses is evaluated first.
so: 
* 'collectie' || 'bundel' => truthy || truthy => true
* true || 'opdracht' => true
* result.administrative_type === true => false

This will always be false

the correct way:
```suggestion
		return ['collectie', 'bundel', 'opdracht'].includes(result.administrative_type);
```